compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1753,6 +1753,7 @@ impl EncodeContext<'a, 'tcx> {
         self.encode_const_stability(def_id);
         self.encode_deprecation(def_id);
         self.encode_item_type(def_id);
+        self.encode_inherent_implementations(def_id);
```

Comment:
we should probably deduplicate the extern item/regular item encoding logic (well, the parts that are shared between them). While I think this would be good to have, you can r=me this PR and just open an E-easy issue for it

Reply:
Made an issue - https://github.com/rust-lang/rust/issues/77393.